ingest-app/docs/api.md

```diff
@@ -561,12 +561,13 @@ Granule metadata can be deleted by sending an HTTP DELETE the URL `%CMR-ENDPOINT
 ## <a name="variable"></a> Variable
 ### <a name="create-update-variable"></a> Create / Update a Variable
 
-Create a UMM-V record and associate that variable to a collection. For associations of other UMM documents, see %CMR-ENDPOINT%/site/docs/search/api.html
+This endpoint is deprecated. To ingest a variable without the collection association please switch to the `%CMR-ENDPOINT%/providers/<provider-id>/variables/<native-id>` API route described below this section.
+
```

Comment:
Is the variable association at ingest meant to be deprecated or is it meant to co-exist with the variable ingest without association? since you provide the option at variable draft publishing...

Reply:
Right now it is co-existing until we communicate with anyone using this endpoint that it is going to go away. Once they move off of it, then we can delete the endpoint.